MeNext/Layout/MainPage.cs

```diff
@@ -24,18 +24,22 @@ public MainPage()
 
 			var homeScreen = new NavigationPage(new HomeScreen());
 			homeScreen.Title = "Home";
+			//homeScreen.Icon = "homeScreenIcon.png";  If we make this icon
```

Comment:
I see no reason not to leave it in. Just leave it commented. The goal is to eventually have icons, even if super low priority.